### PR TITLE
Fix build for other contributors

### PR DESCRIPTION
**Pull request by caraesten**
A few tiny things to make the project easier to build. Super excited for this, thank you SO much for putting it together!

### CODE REVIEW

File: api-gen/gradle.properties [hunk 1]
```diff
@@ -1 +1 @@
-/Users/chr/Documents/ozone/gradle.properties
```
Comment (christiandeange) [inline api-gen/gradle.properties:1]: oops, didn't realize the symlink was an absolute path 😅

File: gradle.properties [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+kotlinVersion=1.8.20
 
-agp.version=7.4.2
+agpVersion=7.4.2
 android.useAndroidX=true
 
-compose.version=1.4.0
+composeVersion=1.4.0
+
+org.gradle.jvmargs=-Xmx8g
```
Comment (christiandeange) [inline gradle.properties:10]: 👍

File: api-gen/gradle.properties (hunk 1 above)
Comment (caraesten) [inline api-gen/gradle.properties:1]: relative symlink :)

**Review by christiandeange — APPROVED** [review]
Thanks for the changes!

**Review by christiandeange — APPROVED** [review]
Thanks for the changes!

File: api-gen/settings.gradle.kts [hunk 3]
```diff
@@ -1,12 +1,13 @@
 pluginManagement {
+  val kotlinVersion: String by settings
```
Comment (caraesten) [inline api-gen/settings.gradle.kts:2]: little nicer to have this typed + with the delegate, plus for some reason, my local builds weren't working w/ the `extra["kotlin.version"]` syntax.

File: gradle.properties (hunk 2 above)
Comment (caraesten) [inline gradle.properties:10]: needed more RAM 🙃

**Comment by caraesten** [comment]
oh dang that was fast, sorry was trying to annotate after you approved! thx!

**Pull request merged**